package.json

```diff
@@ -42,5 +42,8 @@
     "proxyquire": "^2.1.3",
     "sinon": "^9.1.0",
     "sinon-chai": "^3.5.0"
+  },
+  "peerDependencies": {
+    "serverless": "1 || 2"
```

Comment:
Thoughts on `"*"`?

Reply:
Each new major release is not guaranteed to be compatible. So best is to list only versions that are confirmed to be compatible, and whenever a new release happens, add it only after confirming it's compatible.

Concerning v3 of the Framework, once we have v3 candidate, I will confirm that all works, and then prepare PR that will add it here (I'll post this PR before we publish v3)